sycl/test/esimd/regression/windows_msvc_math.cpp

```diff
@@ -0,0 +1,29 @@
+//==------- windows_msvc_math.cpp - DPC++ ESIMD build test -----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: windows
+// RUN: %clangxx -fsycl -fsyntax-only -Xclang -verify %s -I %sycl_include
```

Comment:
What do we need "-I %sycl_include" for?

Reply:
Good point, no need. I copied this boilerplate from another tests. I'll make a change there in a separate patch.